rollup.config.js

```diff
@@ -13,14 +13,10 @@ export default {
         sourcemap: true,
     },
     plugins: [
+        // 'observable-membrane' uses process.env. We don't have that...
+        replace({ "process.env.NODE_ENV": "'production'" }),
         resolve(),
         filesize(),
-        terser({
```

Comment:
Any reason you're dropping this?

The built alpine.js asset is now unminified

Reply:
yeah, it only saves like .1KB

Seems not worth it to me. Unminified means way easier debugging.

Reply:
But it's a good idea to leave it there for when the source code grows bigger. But we can add it later anyways. It's fine.

Reply:
Removing terser shouldn't remove code so not really a breaking change

Have you tried it with mangle: true, seems like that would save a bunch more size by renaming variables.

Reply:
Hi @HugoDF. Thanks for your comment. For some reason, I thought we already had a minified version of the alpine in the "dist" folder. But we actually didn't.

This was the "dist" content before merging this pull request.
https://github.com/alpinejs/alpine/tree/a6496895b4ddea560244c43e4e83407861814a70/dist

JSDeliver supports auto minifying (by adding ".min" before the file extension) so it's not actually needed at all. Sorry for my misleading statement @calebporzio